frappe/public/js/frappe/list/list_view.js

```diff
@@ -952,9 +952,11 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 
 	get_indicator_html(doc) {
 		const indicator = frappe.get_indicator(doc, this.doctype);
+		const title = doc.docstatus == 1 ? 'Document has been submitted' : doc.docstatus == 2 ? 
+			'Document has been cancelled' : 'Document is in draft state';
 		if (indicator) {
 			return `<span class="indicator-pill ${indicator[1]} filterable ellipsis"
-				data-filter='${indicator[2]}'>
+				data-filter='${indicator[2]}' title='${__(title)}'>
```

Comment:
```suggestion
				data-filter='${indicator[2]}' title='${title}'>
```
The actual message should be in translation syntax so that the parser can pick it up.
https://frappeframework.com/docs/user/en/translations#1-literal-strings